src/Sentry.Unity/SentryOptionsUtility.cs

```diff
@@ -11,6 +11,7 @@ public static void SetDefaults(SentryUnityOptions options, IApplication? applica
 
             // 'Optimal' and 'Fastest' don't work on IL2CPP. Forcing 'NoCompression'.
             options.RequestBodyCompressionLevel = CompressionLevelWithAuto.NoCompression;
+            options.EnableAutoSessionTracking = false;
```

Comment:
In the .NET SDK this will be `false` by default until our next major version (to avoid surprising users with 'new data' being sent on a minor update. But here since we're on a experimental version (0.x) we can just default to true:

```suggestion
            options.EnableAutoSessionTracking = true;
```